uelc/main/views.py

```diff
@@ -208,7 +208,10 @@ def notify_facilitators(self, request, path, notification):
                  content=json.dumps(msg))
 
         socket.send(json.dumps(e))
-        socket.recv()
+        try:
+            socket.recv(zmq.NOBLOCK)
+        except zmq.ZMQError:
+            pass
```

Comment:
Yeah, possibly. This error will occur whenever there's no messages for the socket to receive when `socket.recv()` is called.. i'm not familiar enough with everything these notifications are used for in UELC yet, so let me know how you think we should handle this case.